src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/config/MakeRecallDecisionApiExceptionHandler.kt

```diff
@@ -62,6 +63,20 @@ class MakeRecallDecisionApiExceptionHandler {
       )
   }
 
+  @ExceptionHandler(NoDeletedRecommendationRationaleException::class)
+  fun handleNoDeletedRecommendationRationaleException(e: NoDeletedRecommendationRationaleException): ResponseEntity<ErrorResponse> {
+    log.info("Deleted recommendation rationale not found exception: {}", e.message)
+    return ResponseEntity
+      .status(NOT_FOUND)
+      .body(
+        ErrorResponse(
+          status = NOT_FOUND,
+          userMessage = "No deleted recommendation rationale available: ${e.message}",
```

Comment:
I think the principle of the userMessage is that it is something simpler and doesn't expose the actual exception message. Something that might be OK for displaying to a user.